common/app/model/content.scala

```diff
@@ -82,6 +83,7 @@ final case class Content(
   lazy val isHeroic = HeroicTemplateSwitch.isSwitchedOn && tags.isLabourLiverpoolSeries
   lazy val isImmersive = fields.displayHint.contains("immersive") || isImmersiveGallery || tags.isUSMinuteSeries || isHeroic
   lazy val isAdvertisementFeature: Boolean = tags.tags.exists{ tag => tag.id == "tone/advertisement-features" }
+  lazy val campaigns: List[Campaign] = targeting.CampaignAgent.getCampaignsForTags(tags.tags.map(_.id))
```

Comment:
There are no limitations currently, our main issue is we don't know how much the campaign system will be used. We currently have the concept of an 'active' campaign but there's nothing to stop anyone making a super complex campaign with loads of rules which is always active.

The likely outcome is most campaigns will be simple (only a few required/lacking tags). Any advice on latency requirements would be great so I could come up with some sensible limitations.